synapse/storage/databases/main/receipts.py

```diff
@@ -664,6 +664,14 @@ def _insert_linearized_receipt_txn(
             self._receipts_stream_cache.entity_has_changed, room_id, stream_id
         )
 
+        upsert_values: Dict[str, Any] = {
+            "stream_id": stream_id,
+            "event_id": event_id,
+            "data": json_encoder.encode(data),
+        }
+        if stream_ordering is not None:
+            upsert_values["event_stream_ordering"] = stream_ordering
```

Comment:
Any reason to pull this out? If `stream_ordering` is None then it will get inserted as NULL, which is what we'd get anyway?

Reply:
My thinking was the upsert would leave the old value in place if this wasn't set, but I think this is a rare edge case and it's better to have consistency between the columns, so will inline it.

Reply:
https://github.com/matrix-org/synapse/pull/13703/commits/3814eb89371f21c995cb48aa2904bfc47758c457